lib/apiservers/engine/backends/portlayer/container_proxy.go

```diff
@@ -248,6 +274,97 @@ func (c *ContainerProxy) CommitContainerHandle(handle, imageID string) error {
 	return nil
 }
 
+// StreamContainerlogs() reads the log stream from the portlayer rest server and writes
+// it directly to the io.Writer that is passed in.
+func (c *ContainerProxy) StreamContainerLogs(name string, out io.Writer, started chan struct{}, showTimestamps bool, followLogs bool, since int64, tailLines int64) error {
+	defer trace.End(trace.Begin("ContainerProxy.StreamContainerLogs"))
```

Comment:
Just looked at the trace code.  You're right.  Guess, we'll need to remove all those everywhere also.